### DISCUSSION TITLE
How to see trained model's hyperparameters

### DISCUSSION THREAD

**Discussion by thunder-pewp** [autogluon/autogluon#1653] [Q&A] [answered]
After running `predictor = TabularPredictor(label=label, path=save_path).fit(train_data)`

how can we see more details about each model trained?

Not talking about `predictor.info()`.

The `get_info()` and `get_trained_params()` methods of AbstractModel seem like a potential solution, but I don't understand how to use them on the trained models that result from fit().

**Comment by Innixma** [comment, marked as answer]
What information are you looking for? `predictor.info()` provides a lot of information such as per-model hyperparameters, training time, val scores, etc. `predictor.info()` calls the AbstractModel `get_info()` for every model internally and the output is present in the output to `predictor.info()`.

**Reply by thunder-pewp** [reply]
My bad, you're right. I was only seeing a small output and should've opened the full one in a text editor.

On another note, how can I do the same for a multilabel predictor? `multi_predictor.info()` doesn't seem to be the right command.

**Reply by Innixma** [reply]
You'd likely need to get the inner `predictor` for each label in the multilabel predictor. I'm not as familiar with this code so you may need to look into how it is structured. This is a good callout and I'll create an issue for this: https://github.com/awslabs/autogluon/issues/1654

**Reply by thunder-pewp** [reply]
Nice, thanks for your help!
